### ISSUE TITLE
What is your background?

### ISSUE THREAD

**Issue by dmitryvm** [HugoGiraudel/ama#102]

**Comment by HugoGiraudel** [comment]
Hey there,

Thank you for asking! This is a very broad question, so I’m going to do my best to sum up where I come from professionally, and how the hell did I get there.

I’m French. In France, high-school ends at age 17 with the “Bac” exam and diploma. After that, people are free to start working right away, although most choose to do “superior studies”. I summarise, but that’s more or less the typical choice. 

I didn’t know what to do, and I didn’t do much more than playing video games at that point, so I thought I’d try become a game designer of some sort. My parents agreed to get me into Aries, a French school dedicated to 3D development (both for games and animation general). 

The first year was a preparatory class in which I got to learn drawing, painting, sculpting, architecture, history of art, English, as well as some fundamentals of 3D modeling. Unfortunately, that’s right at this moment I realised I really disliked 3D rendering and certainly didn’t want to do that for a living…

Fortunately, Aries also has a web design & development curriculum in 2 years. So after the preparatory class, I joined the first year to learn about infography, product design, packaging design, graphic design and a little bit of web design. If I’m being honest, it wasn’t a particularly interesting year. I remember it being quite boring. But it was a requirement to move to the 2nd year.

So I did, which was entirely dedicated to web development. The interesting thing about this year is that it was done as a work-based cursus: 2 days a week at school, 3 days a week at work. I ended up working in the IT department of the Crédit Agricole bank to help them with web development needs. Not only was I paid (a little) for that, the company also paid for my studies, so my parent didn’t have to. And although the school program already felt a bit out of date (Flash, ActionScript 3, Flex, procedural PHP…), the teacher was really good and kind, so it made learning very enjoyable. I guess I kind of found what I liked. I was going to be a web developer. I aced the final exam with close to the highest score. 

At this point, I was diplomed as a web developer. And at this point, Aries opened a 1-year follow-up to the cursus I just did which was closer to what people know at Computer Science classes. It had unified modeling language theory and practice, object oriented programming fundamentals, C#, C++, Java, network, database, and even some Unity. Crédit Agricole offered to extend my contract for another year so I decided to join this 3rd year.

Aaaaaand it went wrong. I didn’t comfortable there. I had strong disagreement with a few teachers. I disliked basically everything about what we were learning. I was not as good as I was with front-end development and it was frustrating me. Eventually, I failed the exam and decided not to retry it (since they wanted to charge me 1000€ to repass it). 

Crédit Agricole was opening a new affiliate to start an e-commerce platform (which eventually dramatically failed a few years down the road) and offered to hire me as a junior web developer with a very decent starting salary so I accepted.

That’s kind of how I got into web development. :)
